arch/x86/core/intel64/locore.S

```diff
@@ -557,6 +557,7 @@ except: /*
 
 	/* We're done, it's safe to re-enable interrupts. */
 	sti
+	nop
```

Comment:
Following code is just decrementing the stack pointer across the safe area, which is safe as above (RSP must never been seen to change after interrupt handling).

Though unlike above, this is is exception entry and not interrupt entry, which is less latency sensitive and more amenable to conservative "just insert the nop" design.